urdu/urdu-news.md

```diff
@@ -0,0 +1 @@
+if you like to see urdu news [click here](www.bbc.com/urdu)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.